kernel/build/build_deb

```diff
@@ -88,11 +88,17 @@ echo "*****"
 # Clean the git repo
 git clean -fxd
 
+# Install build dependencies
+sudo apt-get install -y bc
+
 # Prep the config
 echo "Preparing the kernel config"
-cp "$(dirname $0)/kernel-config-deb.basic" .config
+wget https://raw.githubusercontent.com/ceph/ceph-build/master/kernel/build/kernel-config-deb.basic -O .config
```

Comment:
It does seem to be a problem we've hit a couple times - "how do I quickly get additional file(s) into the workspace for Jenkins".

Unfortunately I think cloning the ceph-build.git repository is the least-worst option :cry: